engine/class_modules/monk/sc_monk.cpp

```diff
@@ -171,6 +173,15 @@ void monk_action_t<Base>::apply_buff_effects()
   parse_effects( p()->buff.press_the_advantage );
 
   // Conduit of the Celestials
+  parse_effects( p()->buff.flight_of_the_red_crane );
```

Comment:
Flight of the Red Crane doesn't modify actions.
Belongs in `monk_t::parse_player_effects()`